lib/tests/formulae_bottle.rb

```diff
@@ -0,0 +1,386 @@
+# frozen_string_literal: true
+
+module Homebrew
+  module Tests
+    class FormulaeBottle < TestFormulae
+      def run!(args:)
+        each_formulae do |f|
+          bottle!(f, args: args)
+        end
+        deleted_formulae.each do |f|
+          deleted_formula!(f)
+        end
+      end
+
+      def bottle!(formula_name, args:)
+        cleanup_during!(args: args)
+
+        test_header(:FormulaeBottle, method: "bottle!(#{formula_name})")
+
+        built_formulae << formula_name
```

Comment:
We should probably be consistent in whether we use `@` or not-`@` for accessing the attributes.